renv.lock

```diff
@@ -1,6 +1,6 @@
 {
   "R": {
-    "Version": "4.3.2",
+    "Version": "4.3.1",
```

Comment:
surprising that this downgraded. I'm on `4.3.3`, apparently `tanka` is on `4.3.2` and you're on `4.3.1`? I guess it's nbd

Reply:
It should be nbd, but yea, I'm behind.